### ISSUE TITLE
<FieldError case sensitivity can cause confusing use of oAuth provider

### ISSUE THREAD

**Issue by DigitalEntelechy** [redwoodjs/graphql#4547]
Assuming a Firebase Auth setup & given a signup form field like... 
```
            <PasswordField
              className={fieldClasses}
              errorClassName={errorClasses}
              name="Password"
              validation={{ required: true }}
              autoComplete="new-password"
            />
            <FieldError name="Password" className="error m-10" />
```

and a submit function like.... 

```
  const onSubmit = async (data) => {
    signUp({
      displayName: data.name,
      email: data.email,
      password: data.password,
    })
      .catch((error) => {
        console.log('ERROR', errorCode, errorMessage)
      })
  }
  ```
  You will unexpectedly trigger the google oAuth provider workflow when [this line](https://github.com/redwoodjs/redwood/blob/a4e0e927434643aa9d6f713470072c8aa7ed6496/packages/auth/src/authClients/firebase.ts#L36) returns `false` due to `options.password` equaling null. 
  
  Now... why would someone use `Password` & not `password`? 
  
  To control capitalization within the `FieldError` span. 
  
  So.. .code is actually working correctly, but this is a really unexpected outcome for essentially a typo. Def took me a min to sort. 
  
  Perhaps a custom message prop on `FieldError` solves all + increases functionality?

**Comment by Tobbe** [comment]
> Perhaps a custom message prop on `FieldError` solves all + increases functionality?

I haven't thought about any negative implications this might have. But I really like this from a i18n perspective as it lets me keep the strings to be translated in the front-end of my application.

**Comment by BowTiedGar** [comment]
>  I really like this from a i18n perspective as it lets me keep the strings to be translated in the front-end of my application.

Good point! Yes, `children` is probably the natural, eventual outcome. Most flexibility for use cases. 

> That said, we're very busy with V1 preparations right now, so I most likely will not have time to work on this for a while.

Understood! Looking forward to V1!
